etc/cmake/options.cmake

```diff
@@ -108,6 +108,10 @@ if(OTBR_TREL)
     target_compile_definitions(otbr-config INTERFACE OTBR_ENABLE_TREL=1)
 endif()
 
+option(OTBR_EPSKC_ENABLE "Enable ePSKc Support" OFF)
```

Comment:
Per the naming convention, this should be `OTBR_EPSKC`

Reply:
Thanks! Accept.

Reply:
Thanks! Accept.